<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite name="whitefeather.xedge.testcases.Login" tests="14" time="15.417" ignored="0" failures="1" timestamp="13 Sep 2017 07:15:05 GMT" skipped="0" hostname="DESKTOP-TMQM6DS" errors="0">
  <testcase name="verifyUsernameLabel" time="0.069" classname="whitefeather.xedge.testcases.Login"/>
  <testcase name="verifyUsernameInputField" time="0.047" classname="whitefeather.xedge.testcases.Login"/>
  <testcase name="verifyPasswordLabel" time="0.167" classname="whitefeather.xedge.testcases.Login"/>
  <testcase name="verifyLoginPageBackground" time="1.090" classname="whitefeather.xedge.testcases.Login"/>
  <testcase name="verifyLoginButton" time="0.142" classname="whitefeather.xedge.testcases.Login"/>
  <testcase name="enterLoginId" time="0.324" classname="whitefeather.xedge.testcases.Login"/>
  <testcase name="clickLoginButton" time="3.421" classname="whitefeather.xedge.testcases.Login"/>
  <testcase name="verifyPasswordInputField" time="0.047" classname="whitefeather.xedge.testcases.Login"/>
  <testcase name="verifyPasswordWrapper" time="0.138" classname="whitefeather.xedge.testcases.Login"/>
  <testcase name="verifyPasswordPlaceholder" time="0.082" classname="whitefeather.xedge.testcases.Login"/>
  <testcase name="enterLoginPassword" time="0.166" classname="whitefeather.xedge.testcases.Login"/>
  <testcase name="verifyUsernamePlaceholder" time="0.203" classname="whitefeather.xedge.testcases.Login">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.fail(Assert.java:101)
	at whitefeather.xedge.testcases.Login.verifyUsernamePlaceholder(Login.java:88)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:851)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1177)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:756)
	at org.testng.TestRunner.run(TestRunner.java:610)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:387)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:382)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1293)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1218)
	at org.testng.TestNG.runSuites(TestNG.java:1133)
	at org.testng.TestNG.run(TestNG.java:1104)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
    </failure>
  </testcase> <!-- verifyUsernamePlaceholder -->
  <testcase name="openLoginPage" time="9.436" classname="whitefeather.xedge.testcases.Login"/>
  <testcase name="verifyClientAccountLogo" time="0.085" classname="whitefeather.xedge.testcases.Login"/>
</testsuite> <!-- whitefeather.xedge.testcases.Login -->
